js/blankView.js

```diff
@@ -8,7 +8,7 @@ export default class BlankView extends ComponentView {
 
   postRender() {
     this.setReadyStatus();
-    this.setupInviewCompletion();
+    this.setCompletionStatus();
```

Comment:
```suggestion
    this.setupInviewCompletion();
```

Reply:
Can we leave this as it was, as the `_isOptional: false` + inview can be useful to auto complete on page end?